client/src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from "react";
```

Comment:
Since the LoginPage and the SignUpPage are pretty much the same, I think we can consolidate them into one and have props where they'll just changed based on:
- The text that we need to provide
- The onClick handlers that we need
- etc.

Reply:
Fixed: Combined `SignUpPage` and `LoginPage` implementation into `CredentialsPage`. The appropriate props are passed in `SignUpPage` and `LoginPage` respectively. Each page still has its own file.